src/pasteld.cpp

```diff
@@ -176,12 +183,70 @@ bool AppInit(int argc, char* argv[])
     return fRet;
 }
 
+#ifdef __linux__
+void print_callstack()
+{
+    const int max_frames = 64;
+    void* addrlist[max_frames];
+
+    int addrlen = backtrace(addrlist, sizeof(addrlist) / sizeof(void*));
+
+    if (addrlen == 0) {
+        LogFnPrintf("No stack trace found");
+        return;
+    }
+
+    char** symbol_list = backtrace_symbols(addrlist, addrlen);
+    auto guard = sg::make_scope_guard([&]() noexcept 
+    {
+        free(symbol_list);
+    });
+
+    LogFnPrintf("Stack trace:");
+    for (int i = 1; i < addrlen; i++)
+    {
+        LogFnPrintf("%s", symbol_list[i]);
+        Dl_info info;
+        if (dladdr(addrlist[i], &info) && info.dli_fbase)
+        {
+            void* offset = (void*)((uintptr_t)addrlist[i] - (uintptr_t)info.dli_fbase);
+            char syscom[256];
+            snprintf(syscom, sizeof(syscom), "addr2line %p -e %s", offset, info.dli_fname);
+            system(syscom);
+        }
+    }
+}
+#else
+void print_callstack()
+{
+    return;
+}
+#endif
+
+void pasteld_terminate()
+{
+    LogFnPrintf("pasteld_terminate():");
+#ifdef __linux__
```

Comment:
don't need it here, i mean #ifdef

Reply:
yep, I've added dummy print_callstack for platforms other than linux